lib/test.js

```diff
@@ -108,12 +108,12 @@ function Test (options) {
 
   // Bind all methods.
   var bound = {}
-  for (var m in this) {
+  Object.keys(this).forEach(function (m) {
```

Comment:
Semantic change.  The intent here is to bind inherited as well as local functions.  (Generally, all or most methods are properties of the prototype, not local properties.)